kubernetes-elasticsearch-cluster/deploy-es.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
I changed es-data-stateful.yaml (increase disk size), reran deploy-es.sh, and expected to see the new setting. The new setting did not take effect. Please make whatever changes are necessary to have new settings take effect -- maybe kubectl delete.